integration_tests/coverage-remapping-inline-sourcemap/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "jest": {
+    "rootDir": "./",
+    "transform": {
+      "^.+\\.(js)$": "babel-jest"
```

Comment:
I don't think this test is valuable currently because babel-jest doesn't deal with source maps. It instruments and then transforms, so there's no coverage remapping necessary.

Reply:
Doesn't it emit inline sourcemaps? [Here](https://github.com/facebook/jest/blob/97ee7c985bcafdc0ac203ba443a5fb791fbc3b5d/packages/babel-jest/src/index.js#L73) Which are then read?

Reply:
I'd also like to better understand this 🙂

Reply:
FWIW, inline sourcemaps is pretty recent: #3738

Reply:
Since mapCoverage is not passed in this integration test (which is correct*), no remapping should be performed here. I'm not positive, but I would expect this integration test to pass with the same output on master. Since the main purpose of this PR, as I understand it, is accurate line/col numbers for error stacktraces and codeframes, maybe the integration test should try to cover one of those use cases.

* If a transformer instruments before doing downlevel compilation (as babel-jest does), any sort of source map processing for coverage is probably going to break things. Source map processing for coverage only makes sense when going the other direction (downlevel compilation, then instrumentation)